defaults/main.yml

```diff
@@ -57,7 +57,9 @@ datadog_skip_running_check: false
 datadog_agent_allow_downgrade: no
 
 # default datadog windows download url
-datadog_windows_download_url: "https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-6-latest.amd64.msi"
+# we're pinning 6.14.2 instead of latest for now. See http://dtdg.co/win-614-fix for more details.
+#datadog_windows_download_url: "https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-6-latest.amd64.msi"
```

Comment:
remove the commented out line?

Reply:
Keeping it makes it easier to revert to latest.